front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Editor/RenderNode/index.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useEffect } from 'react';
+
+import { Box, colors } from '@citizenlab/cl2-component-library';
+import { useNode, useEditor, ROOT_NODE } from '@craftjs/core';
+import styled from 'styled-components';
+
+import messages from 'components/admin/ContentBuilder/messages';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import { WIDGET_TITLES, hasNoPointerEvents } from '../../CraftComponents';
+
+const StyledBox = styled(Box)`
+  ${({ isRoot }: { isRoot: boolean }) =>
+    isRoot
+      ? `cursor: auto;
+          padding: 4px;
+          width: 100%;
+          max-width: 1000px;
+          background-color: #fff;
+          min-height: 160px;`
+      : `cursor:move;`}
+`;
+
+const CONTAINER = 'Container';
+
+const RenderNode = ({ render }) => {
+  const {
+    id,
+    name,
+    isHover,
+    hasError,
+    title,
+    noPointerEvents,
+    connectors: { connect, drag },
+  } = useNode((node) => {
+    // This can sometimes be undefined, even though
+    // craftjs says it can't
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    if (!node) return {};
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    if (!node.data) return {};
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    if (!node.events) return {};
+
+    const name = node.data.name;
+
+    return {
+      props: node.data.props,
+      isHover: node.events.hovered,
+      name,
+      hasError: node.data.props.hasError,
+      title: WIDGET_TITLES[name],
```

Comment:
Before, this was also shared with the project description builder. Because we now use these centrally defined titles and pointer events I created a new version.